### PR TITLE
docs(parquet): Parquet loader docs

### PR DESCRIPTION
**Pull request by ibgreen**

### CODE REVIEW

File: modules/parquet/docs/api-reference/parquet-loader.md [hunk 1]
```diff
@@ -1,14 +1,23 @@
 # ParquetLoader
 
+<p class="badges">
+  <img src="https://img.shields.io/badge/From-v2.1-blue.svg?style=flat-square" alt="From-v3.1" />
+	<img src="https://img.shields.io/badge/-BETA-teal.svg)](/studio/user-guide/import" alt="BETA">
```
Comment (kylebarron) [inline modules/parquet/docs/api-reference/parquet-loader.md:5]:
```suggestion
  <img src="https://img.shields.io/badge/-BETA-teal.svg)](/studio/user-guide/import" alt="BETA">
```
Nit: Mixed tabs/spaces

File: modules/parquet/docs/api-reference/parquet-loader.md [hunk 2]
```diff
@@ -1,14 +1,23 @@
 # ParquetLoader
 
+<p class="badges">
+  <img src="https://img.shields.io/badge/From-v2.1-blue.svg?style=flat-square" alt="From-v3.1" />
```
Comment (kylebarron) [inline modules/parquet/docs/api-reference/parquet-loader.md:4]:
```suggestion
  <img src="https://img.shields.io/badge/From-v3.1-blue.svg?style=flat-square" alt="From-v3.1" />
```

File: modules/parquet/docs/README.md [hunk 3]
```diff
@@ -1,142 +1,97 @@
...
-| `SNAPPY` |YES| YES|
-| `GZIP` |YES| YES|
-| `LZO` |NO| NO|
-| `BROTLI` |NO| YES|
-| `LZ4`  |YES| YES|
-| `ZSTD` |YES| YES|
-| `LZ4_RAW` |YES| YES|
+<p class="badges">
+  <img src="https://img.shields.io/badge/From-v2.1-blue.svg?style=flat-square" alt="From-v3.1" />
+	<img src="https://img.shields.io/badge/-BETA-teal.svg)](/studio/user-guide/import" alt="BETA">
```
Comment (kylebarron) [inline modules/parquet/docs/README.md:5]:
```suggestion
  <img src="https://img.shields.io/badge/From-v3.1-blue.svg?style=flat-square" alt="From-v3.1" />
  <img src="https://img.shields.io/badge/-BETA-teal.svg)](/studio/user-guide/import" alt="BETA">
```

File: modules/parquet/docs/api-reference/parquet-writer.md [hunk 4]
```diff
@@ -1 +1,11 @@
-# ParquetWriter
\ No newline at end of file
+# ParquetWriter
+
+<p class="badges">
+  <img src="https://img.shields.io/badge/From-v2.1-blue.svg?style=flat-square" alt="From-v3.1" />
+	<img src="https://img.shields.io/badge/-BETA-teal.svg)](/studio/user-guide/import" alt="BETA">
```
Comment (kylebarron) [inline modules/parquet/docs/api-reference/parquet-writer.md:5]:
```suggestion
  <img src="https://img.shields.io/badge/From-v3.1-blue.svg?style=flat-square" alt="From-v3.1" />
  <img src="https://img.shields.io/badge/-BETA-teal.svg)](/studio/user-guide/import" alt="BETA">
```

**Review by kylebarron — APPROVED** [review]

File: modules/parquet/docs/README.md (hunk 3 above)
Comment (morphoix) [inline modules/parquet/docs/README.md:5]: alt="BETA" />

File: modules/parquet/docs/api-reference/parquet-writer.md (hunk 4 above)
Comment (morphoix) [inline modules/parquet/docs/api-reference/parquet-writer.md:5]: alt="BETA" />

**Comment by kylebarron** [comment]
> alt="BETA" />

Looks like the main issue is the mix of html and markdown image tags

**Pull request merged**